### PR TITLE
Update XenToOvirt.html.md

### PR DESCRIPTION
**Pull request by shaharha**
Added guideline for importing VMs with block disks

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)
needs review by @ahadas

### CODE REVIEW

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 1]
```diff
@@ -34,3 +34,11 @@ We are logging to the Xen host after we copy the ssh key in order to check that
 - Select **'XEN (via EL)'** in the source select box
 - Select VDSM host from the 'Proxy Host' select box
 - Enter valid URI such as: `xen+ssh://user@xenhost`
+
+## Import VMs with Block disks
+Currently it is not possible to import Xen VMs with block disks via the oVirt site.
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: oVirt site? should be webadmin

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 2]
```diff
@@ -34,3 +34,11 @@ We are logging to the Xen host after we copy the ssh key in order to check that
 - Select **'XEN (via EL)'** in the source select box
 - Select VDSM host from the 'Proxy Host' select box
 - Enter valid URI such as: `xen+ssh://user@xenhost`
+
+## Import VMs with Block disks
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: Block disks or Block devices? not sure what is the right terminology

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 3]
```diff
@@ -34,3 +34,11 @@ We are logging to the Xen host after we copy the ssh key in order to check that
...
+Currently it is not possible to import Xen VMs with block disks via the oVirt site.
+As a workaround please follow these steps:
+- Make sure you have export domain active
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: Make sure you have an active export domain

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 4]
```diff
@@ -34,3 +34,11 @@ We are logging to the Xen host after we copy the ssh key in order to check that
...
+- Run the command: `virt-v2v-copy-to-local -ic xen+ssh://root@xenserver.com vmname`
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: should we maybe write it as 
`virt-v2v-copy-to-local -ic xen+ssh://<root>@<xenserver.com> <vmname>`
to make it clear that the user needs to replace it with the specific properties in this environment?

File: source/develop/release-management/features/virt/XenToOvirt.html.md (hunk 2 above)
Comment (shaharha) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: Block disks

File: source/develop/release-management/features/virt/XenToOvirt.html.md (hunk 1 above)
Comment (shaharha) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: done

**Pull request merged**